test/fuzz/basic.ml

```diff
@@ -91,7 +91,11 @@ let ibinop =
     ; const (Rotr : ibinop)
     ]
 
-let iunop = choose [ const Clz; const Ctz; const Popcnt ]
+(* TODO :
+   temp comment for symbolic context "const Popcnt"
+   iunop_popcnt_32 / iunop_popcnt_64 in concrete contexte
+*)
+let iunop = choose [ const Clz; const Ctz (*; const Popcnt*) ]
```

Comment:
Could you put `popcnt` back as it was before ?

Reply:
gen.ml :
```ocaml
if env.Env.conf = Concrete then
        [ pair B.extend_32_i32 (const [ S.Nothing ])
        ; pair B.i64_extend_i32 (const [ S.Pop; S.Push (Num_type I64) ])
        ; pair B.iunop_popcnt_32 (const [ S.Nothing ])
        ]
      else []
```

Reply:
I know it is not implemented, but I would like to get this PR ready and fix popcount implementation later.